Peer/Verbs/DetailsOptions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CommandLine;
+
+namespace Peer.Verbs
+{
+    [Verb("details", isDefault: false, HelpText = "Show additional details about a PullRequest")]
+    public class DetailsOptions
+    {
+        [Value(0, Required = true,
+        HelpText = "A full or partial pull request identifier of the form provider/org/repo/id" +
+        " partials can be built up from combinations of sections. For example: '6' to get" +
```

Comment:
Where partials are used in more than one place it might make sense to have a global helptext that explains them and just reference it in the commands that rely on them.

Reply:
Yeah that makes sense I'll do that

Reply:
Done

Reply:
done